e2e/Sandbox/Reveal/AuthenticationProvider.cs

```diff
@@ -153,5 +155,13 @@ internal string RetrieveGoogleDriveBearerToken()
 
             return accessToken;
         }
+
+        async Task<string> CreateJwtToken()

```

Comment:
Check the function RetrieveGoogleDriveBearerToken, create the function similar to it (for consistency), or create a common function to retreive token by Json data and a list of scopes.

Reply:
updated